app/models/game.rb

```diff
@@ -29,40 +29,40 @@ def populate_board
-    @board[6][2] = Pawn.create(:color => :black, :name => :pawn, :position => "26", :x_axis => 2, :y_axis => 6)
-    @board[6][3] = Pawn.create(:color => :black, :name => :pawn, :position => "36", :x_axis => 3, :y_axis => 6)
-    @board[6][4] = Pawn.create(:color => :black, :name => :pawn, :position => "46", :x_axis => 4, :y_axis => 6)
-    @board[6][5] = Pawn.create(:color => :black, :name => :pawn, :position => "56", :x_axis => 5, :y_axis => 6)
-    @board[6][6] = Pawn.create(:color => :black, :name => :pawn, :position => "66", :x_axis => 6, :y_axis => 6)
-    @board[6][7] = Pawn.create(:color => :black, :name => :pawn, :position => "76", :x_axis => 7, :y_axis => 6)
+    @board[7][0] = Rook.create(:color => :black, :name => :rook, :position => "07", :x_axis => 0, :y_axis => 7, :image => 'black-rook.gif')
+    @board[7][1] = Knight.create(:color => :black, :name => :knight, :position => "17", :x_axis => 1, :y_axis => 7, :image => 'black-knight.gif')
+    @board[7][2] = Bishop.create(:color => :black, :name => :bishop, :position => "27", :x_axis => 2, :y_axis => 7, :image => 'black-bishop.gif')
+    @board[7][3] = Queen.create(:color => :black, :name => :queen, :position => "37", :x_axis => 3, :y_axis => 7, :image => 'black-queen.gif')
+    @board[7][4] = King.create(:color => :black, :name => :king, :position => "47", :x_axis => 4, :y_axis => 7, :image => 'black-king.gif')
+    @board[7][5] = Bishop.create(:color => :black, :name => :bishop, :position => "57", :x_axis => 5, :y_axis => 7, :image => 'black-bishop.gif')
+    @board[7][6] = Knight.create(:color => :black, :name => :knight, :position => "67", :x_axis => 6, :y_axis => 7, :image => 'black-knight.gif')
+    @board[7][7] = Rook.create(:color => :black, :name => :rook, :position => "77", :x_axis => 7, :y_axis => 7, :image => 'black-rook.gif')
+    @board[6][0] = Pawn.create(:color => :black, :name => :pawn, :position => "06", :x_axis => 0, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][1] = Pawn.create(:color => :black, :name => :pawn, :position => "16", :x_axis => 1, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][2] = Pawn.create(:color => :black, :name => :pawn, :position => "26", :x_axis => 2, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][3] = Pawn.create(:color => :black, :name => :pawn, :position => "36", :x_axis => 3, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][4] = Pawn.create(:color => :black, :name => :pawn, :position => "46", :x_axis => 4, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][5] = Pawn.create(:color => :black, :name => :pawn, :position => "56", :x_axis => 5, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][6] = Pawn.create(:color => :black, :name => :pawn, :position => "66", :x_axis => 6, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][7] = Pawn.create(:color => :black, :name => :pawn, :position => "76", :x_axis => 7, :y_axis => 6, :image => 'black-pawn.gif')
```

Comment:
sorry I am not sure what you mean. The :image field exists within the pieces database already in my schema. I think that was already there when the database was created before.